content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md

```diff
@@ -387,6 +387,110 @@ Allows fine-grained controls of various features only available in the enterpris
+        * *Default:* 0
+
+    - `preGeneratedKeysTopUpInterval`
+
+        * Defines the time interval (in seconds) at which the pre-generated keys are topped up or refreshed.
+        * *Default:* 300 seconds
+
+    - `recoveryMaximumBackupInterval`
+
+        * Specifies the maximum time duration for which ledger recovery can be done. This parameter can be specified in
+        three different ways:
+          * Network parameters (if node is on a network using CENM 1.6 or later)
+          * Configuration parameter (this parameter)
+          * Directly as a parameter on the recovery flow
+
+        * If the parameter is specified in more than one place, then the recovery flow parameter overrides the configuration
+        parameter which overrides the network parameter. If the node is on a network not using CENM 1.6 and the parameter
+        is not specified in either configuration or directly on the ledger recovery flow, then an exception is raised
+        when starting the recovery flow.
+        * There is no default for this parameter. If it is not specified in any of the above
+        ways, an exception is raised.
+        * The duration is specified in the Human-Optimized Config Object Notation (HOCON) format with suffixes of `h` (hours),
+        `m` (minutes) and `s` (seconds), for example, `1h` means one hour. For additional information on the HOCON duration
+        format parsing, see the [HOCON duration format file](https://github.com/lightbend/config/blob/master/HOCON.md) on GitHub.
+        * *Default:* null
+
+    - `confidentialIdentityMinimumBackupInterval`
+
+        * Defines the time interval (duration) within which automatic database backups occur, ensuring that all pre-generated
+        keys created before the backup are marked as backed-up according to this interval. This parameter can be specified
+        in two different ways:
+          * Network parameters (if node is on a network using CENM 1.6 or later)
+          * Configuration parameter (this parameter)
+
+        * If the parameter is specified in more than one place, then the configuration parameter overrides the network parameter.
+        * If the node is on a network not using CENM 1.6 and the parameter is not specified in configuration, then an exception
+        is raised if either `noOfPreGeneratedKeys` or `noOfPreGeneratedKeysWithCerts` is greater than 0. In other words, if
+        you want to regenerate keys, then you must have the `confidentialIdentityMinimumBackupInterval` defined. There is
+        no default for this parameter.
+        * The duration is specified in the Human-Optimized Config Object Notation (HOCON) format with suffixes of `h` (hours),
+        `m` (minutes) and `s` (seconds), for example, `1h` means one hour. For additional information on the HOCON duration
+        format parsing, see the [HOCON duration format file](https://github.com/lightbend/config/blob/master/HOCON.md) on GitHub.
+        * *Default:* null
```

Comment:
If either noOfPreGeneratedKeys or noOfPreGeneratedKeysWithCerts is greater than zero then confidentialIdentityMinimumBackupInterval must be set by one of the ways described above.

Reply:
Updated

Reply:
I meant to replace Default: null with:

Default: If either noOfPreGeneratedKeys or noOfPreGeneratedKeysWithCerts is greater than zero then confidentialIdentityMinimumBackupInterval must be set by one of the ways described above.